packages/x-date-pickers-pro/src/internals/hooks/useEnrichedRangePickerFieldProps.ts

```diff
@@ -195,15 +195,15 @@ const useMultiInputFieldSlotProps = <
   }, [rangePosition, open, currentView, startFieldRef, endFieldRef]);
 
   const openRangeStartSelection: React.UIEventHandler = (event) => {
-    event.stopPropagation();
+    event.preventDefault();
```

Comment:
These fixes prevent the following issue:

https://github.com/mui/mui-x/assets/4941090/bb72798d-ad8e-4cdb-895e-26e4731714fb

The regular Picker is not closed when opening a range picker

Reply:
Could we add a comment on top of this preventDefault to explain why it's here? In general, I think we should aim to justify each, they (preventDefault and especially stopPropagation) tend to be quickly food guns.